tests/sentry/models/test_organization.py

```diff
@@ -39,6 +39,18 @@
 
 
 class OrganizationTest(TestCase):
+    def test_slugify_on_new_orgs(self):
+        org = Organization.objects.create(name="name", slug="---downtown_canada---")
+        assert org.slug == "downtown-canada"
+
+        # Only slugify on new instances of Organization
+        org.slug = "---downtown_canada---"
+        org.save()
```

Comment:
after save of model, do you need to refetch the model from the db?
Otherwise we haven't asserted the db has properly updated the slug?

Reply:
I added `org.refresh_from_db()` just in case.